### PR TITLE
Add packages for various Python modules needed for twisted tls support

### PR DESCRIPTION
**Pull request by fred-wang**
Here are more packages for the tls support needed by twisted/buildbot (cf #893)
However, to call crypto libraries we will need the cffi package too (cf #889).

### CODE REVIEW

File: mingw-w64-python-ipaddress/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Contributor: Frederic Wang <fred.wang@free.fr>
+
+_realname=ipaddress
+pkgbase=mingw-w64-python-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-python2-${_realname}" "${MINGW_PACKAGE_PREFIX}-python3-${_realname}")
+pkgver=1.0.15
+pkgrel=1
+pkgdesc='Port of the 3.3+ ipaddress module to 2.6, 2.7, 3.2 (mingw-w64)'
```
Comment (Alexpux) [inline mingw-w64-python-ipaddress/PKGBUILD]: Our python3 is 3.5. Do we need this for our Python3?

File: mingw-w64-python-pyasn1-modules/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+pkgver=0.0.8
+pkgrel=1
+pkgdesc='A collection of ASN.1-based protocols modules (mingw-w64)'
+url='http://sourceforge.net/projects/pyasn1/'
+license=('BSD')
+arch=('any')
+makedepends=("${MINGW_PACKAGE_PREFIX}-python2"
+             "${MINGW_PACKAGE_PREFIX}-python3"
+             "${MINGW_PACKAGE_PREFIX}-python2-pyasn1"
+	     "${MINGW_PACKAGE_PREFIX}-python2-pyasn1")
```
Comment (Alexpux) [inline mingw-w64-python-pyasn1-modules/PKGBUILD]: here seems you need python3

**Comment by fred-wang** [comment]
@Alexpux I addressed the review comments.

File: mingw-w64-python-ipaddress/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Maintainer: Frederic Wang <fred.wang@free.fr>
+
+_realname=ipaddress
```
Comment (Alexpux) [inline mingw-w64-python-ipaddress/PKGBUILD]: Rename folder with this PKGBUILD to match python2 only

**Comment by fred-wang** [comment]
@Alexpux done

File: mingw-w64-python2-ipaddress/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+package_python2-ipaddress() {
+  cd ${srcdir}/build-python2
+  ${MINGW_PREFIX}/bin/python2 setup.py build
+  MSYS2_ARG_CONV_EXCL="--prefix=;--install-scripts=;--install-platlib=" \
+  ${MINGW_PREFIX}/bin/python2 setup.py install --prefix=${MINGW_PREFIX} --root="${pkgdir}"
+}
+
+package_mingw-w64-i686-python2-ipaddress() {
```
Comment (Alexpux) [inline mingw-w64-python2-ipaddress/PKGBUILD]: Remove this functions. You need here only regular "build" and "package" functions.

**Comment by fred-wang** [comment]
@Alexpux I hope that this latest version is good. However, I haven't tested it again on a Windows system.

**Pull request merged**